jobs/pacman-rule-engine-2.0/src/main/java/com/tmobile/pacman/util/AuditUtils.java

```diff
@@ -65,7 +65,7 @@ public static void postAuditTrail(List<Annotation> annotations, String status) {
                     try {
                         CommonUtils.doHttpPost(requestUrl, requestBody.toString());
                     } catch (Exception e) {
-                        logger.error("Audit creation failed", e);
+                        logger.error("Error occurred in job with policyUUID:  Error message - Audit creation failed, ", e);

```

Comment:
consider extracting this to a constant and standardizing the errors per same comments as for the collectors

Reply:
done.